src/hypersync/RestDataSource.ts

```diff
@@ -389,6 +393,7 @@ export class RestDataSource extends ApiClient implements IDataSource {
         const response = await this.getData<any>(
           lookup.dataSet,
           params,
+          undefined,
```

Comment:
The asymmetry here (`page` is always undefined but `metadata` may not be) is kind of bugging me.  But maybe it all makes sense.  We're saying that we don't support paging when applying lookups, but we do support the pending business--i.e. I could trigger a long running data gen process in some service and when it is done I could use `metadata` to pull out the base data set as well as one or more lookups.

If that all makes sense to you then I'm fine with this change.  But I wouldn't mind a `// Paging is not supported in lookups` comment after the `undefined`.

Reply:
Yeah, exactly. It didn't make any sense to me that we would fetch a value and use the request's page value. If we were collecting members in multiple groups, for instance, the `page` value would be for the list of groups, and the lookup would be the users. We wouldn't pass `page` into that `getData` call.

I'll be sure to comment